dart/test/sentry_test.dart

```diff
@@ -102,7 +102,7 @@ void main() {
     });
   });
 
-  test("options can't be null", () {
+  test("options can't be null", () async {
```

Comment:
is async still needed ?

Reply:
guess not, not sure why I've added either, removed it.